apps/src/componentLibrary/button/types.ts

```diff
@@ -0,0 +1,8 @@
+export type ButtonType =
+  | 'primary'
+  | 'secondary'
+  | 'tertiary'
+  | 'iconBorder'
+  | 'iconOnly';
```

Comment:
Should these be two different dimensions actually (`primary | secondary | tertiary` and `icon | text`)? Couldn't primary/secondary apply to both text buttons and icon buttons? Looking at the storybook, it looks we use iconBorder and iconOnly to set the border for icon buttons, but could we accomplish that with primary/secondary/tertiary instead?

Reply:
From what I've seen in Figma:
`'primary' | 'secondary' | 'tertiary' ` - can be text
`'iconBorder' | 'iconOnly';` - can be icon only

Will doublecheck with design team though, good question, thank you!